packages/paste-website/src/pages/utilities/aspect-ratio/index.mdx

```diff
@@ -0,0 +1,172 @@
+displays of various sizes. For example, you can use Aspect Ratio when a
+customer needs to see a preview of an uploaded document.
+
+Aspect Ratio can also be used to embed components, or other HTML elements that
+needs to be set to a specific height-to-width ratio. A component or other HTML
+element use case is the need to align an HTML element with an image in two columns.
+See examples.
+
+Another use case is for color swatches on a documentation site. These color swatches
+could be a specific height/width div that need to resize on specific devices.
+Aspect Ratio will allow the color swatches to resize correctly based on a ratio.
+
+<DoDont>
+  <Do title="Do" body="Use for an image or embedded video, and have it resize at a specific ratio." preview={false} />
+  <Dont title="Don't" body="Don’t use with fixed size (width & height) elements, as those will not adhere to the height-to-width ratio." preview={false} />
+</DoDont>
+
+<DoDont>
+  <Do title="Do" body="Use to embed a component or other HTML element." preview={false} />
+  <Dont title="Don't" body="Don’t use without any child elements, because this is only a container element." preview={false} />
+</DoDont>
+
+### Examples
+
+**Image and embedded component use case:**<br/>
+(Using an Anchor component do/don't for this example).
+
+<DoDont>
+  <Do title="" body="" center>
+    <Anchor href="https://paste.twilio.design">Go to API documentation</Anchor>
+  </Do>
+  <Dont title="" body="">
+    <Img fluid={props.data.file.childImageSharp.fluid} />
+  </Dont>
+</DoDont>
+
+**Color swatch use case:**
+
+<LivePreview scope={{Absolute, AspectRatio, Box}} language="jsx">
+  {`<Box
+  padding="space30"
+  maxWidth="size50"
+  borderColor="colorBorderLight"
+  borderStyle="solid"
+  borderWidth="borderWidth10"
+>
+  <AspectRatio ratio="4:3">
+    <Absolute backgroundColor="colorBackgroundBrand" />
+  </AspectRatio>
+</Box>`}
+</LivePreview>
+
+### Anatomy
+
+No tokens are used.
+
+</content>
+
+***
+
+<content>
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/anchor
+```
+
+#### Usage
+
+```js
+import {AspectRatio} from '@twilio-paste/aspect-ratio';
+
+<AspectRatio ratio="4:3">
+  ... Embedded element
+</AspectRatio>
+```
+
+#### Storybook
+
+[View Aspect Ratio in Storybook](https://twilio-labs.github.io/paste/?path=/story/utilities-aspect-ratio--default)
+
+
+#### Props
+
+| Prop           | Type                  | Description                                                                                                              | Default   |
+| -------------- | --------------------- | ------------------------------------------------------------------------------------------------------------------------ | --------- |
+| ratio?         | string                | Determines the aspect ratio of the element. Use a colon separated number pattern (width:height). Required                | 4:3       |
```

Comment:
If this is required why is there a `?` near ratio? 
Maybe the `?` should be next to string.

Reply:
`?` was a failed copy/paste and not looking closely at it.